src/components/mainPage/Header.vue

```diff
@@ -0,0 +1,77 @@
+<template>
+  <div class="header">
+    <div class="header-bg"></div>
+    <div class="header-content">
+      <div class="header-logo">
+        <Logo firstPart="netflix" secondPart="roulette" />
+      </div>
+      <div class="search-header">FIND YOUR MOVIE</div>
+      <div class="search-bar"><SearchBar /></div>
+      <div class="search-by">
+        <FilterButtons
+          message="SEARCH BY"
+          firstButtonTitle="TITLE"
+          secondButtonTitle="GENRE"
+          firstOption="title"
```

Comment:
Under "src/core" folder that you create after one of the above comments consider creating a folder "constants" that will hold any number of file to your liking - as long as they contain all the hard-coded numbers/strings that you use throughout the app.
The idea is to have a central place for all the constants to be able to easily reuse/change them.

Reply:
Fixed